packages/openactive-openid-test-cli/lib/cli.js

```diff
@@ -83,29 +84,26 @@ const argv = yargs(process.argv.slice(2))
       type: 'string',
       alias: 'u',
       description: 'Username of the Seller',
-      default: 'test',
+      default: 'test1',
     },
     password: {
       type: 'string',
       alias: 'p',
       description: 'Password of the Seller',
-      default: 'test',
+      default: 'test1',
     },
     usernameFieldSelector: {
       type: 'string',
-      alias: 'l',
```

Comment:
These all used the same alias and this caused all of the selectors to be `[name='username' i]`, which lead to CLI failing